src/NLog.Web/README.md

```diff
@@ -0,0 +1,30 @@
+# NLog.Web
+
+- Logger Provider for ASP.NET platform.
+- Log all properties of HttpContext and HostingEnvironment via [layout renderers](https://nlog-project.org/config/?tab=layout-renderers&search=package:nlog.web) just by changing the NLog.config file.
```

Comment:
"Extends logging output with details from the active HttpContext via layout renderers, just by changing NLog.config file."

Reply:
Completed